view/player/tmpl/default.html.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Nooku Framework - http://nooku.org/framework
+ *
+ * @copyright	Copyright (C) 2011 - 2014 Johan Janssens and Timble CVBA. (http://www.timble.net)
+ * @license		GNU GPLv3 <http://www.gnu.org/licenses/gpl.html>
+ * @link		http://github.com/nooku/nooku-files for the canonical source repository
+ */
+defined('KOOWA') or die;
+?>
+
+<?= helper('behavior.koowa'); ?>
+
+<ktml:style src="assets://files/css/plyr.css" />
+<ktml:script src="assets://files/js/plyr/plyr.js" />
+<script>
```

Comment:
@johanjanssens they were originally included using JDocument but per discussion we moved them into a layout. They need to be included to the page headers and so far the only ways that I know of are either JDocument or khtml tags in a layout. Is there a koowa way of including them in headers within the load function instead of importing a template layout?